src/mudlet.cpp

```diff
@@ -3015,32 +3015,37 @@ void mudlet::toggleMute(bool state, QAction* toolbarAction, QAction* menuAction,
         mpActionMuteGame->setIcon(QIcon(mMuteGame ? qsl(":/icons/unmute.png") : qsl(":/icons/mute.png")));
     }
 
-    // Toolbar icon. "Mute" when any protocol is unmuted. "Unmute" only when all protocols are muted.
+    // Toolbar icon. "Mute all media" when any protocol is unmuted. "Unmute all media" only when all protocols are muted.
     const bool isMediaMuted = mediaMuted();
     mpActionMuteMedia->setIcon(QIcon(isMediaMuted ? qsl(":/icons/unmute.png") : qsl(":/icons/mute.png")));
     mpActionMuteMedia->setText(isMediaMuted ? tr("Unmute all media") : tr("Mute all media"));
     mpActionMuteMedia->setChecked(isMediaMuted);
     dactionMuteMedia->setChecked(isMediaMuted);
-    mpButtonMute->setText(isMediaMuted ? tr("Unmute") : tr("Mute"));
-    mpButtonMute->setChecked(false);
+    mpButtonMute->setText(isMediaMuted ? tr("Unmute all media") : tr("Mute all media"));
+    mpButtonMute->setChecked(isMediaMuted);
     mpButtonMute->setEnabled(true);
 
     // Notify when all media is muted or all media is unmuted. Helps if the shortcut is hit accidentally.
     if (isMediaMuted || mediaUnmuted()) {
         QString message;
 
         for (auto pHost : mHostManager) {
-            const QKeySequence* sequence = pHost->profileShortcuts.value(qsl("Mute all media"));
+            if (mudlet::self()->showMuteAllMediaTutorial()) {
+                const QKeySequence* sequence = pHost->profileShortcuts.value(qsl("Mute all media"));
 
-            if (sequence && !sequence->toString().isEmpty()) {
-                message = isMediaMuted
-                    ? tr("[ INFO ]  - Mudlet and game sounds are muted. Use %1 to unmute.").arg(sequence->toString())
-                    : tr("[ INFO ]  - Mudlet and game sounds are unmuted. Use %1 to mute.").arg(sequence->toString());
-            } else {
-                message = isMediaMuted ? tr("[ INFO ]  - Mudlet and game sounds are muted.") : tr("[ INFO ]  - Mudlet and game sounds are unmuted.");
-            }
+                if (sequence && !sequence->toString().isEmpty()) {
+                    const QString seq = sequence->toString(QKeySequence::NativeText).split("", Qt::SkipEmptyParts).join(">+<");
+
+                    message = isMediaMuted
+                        ? tr("[ INFO ]  - Mudlet and game sounds are muted. Use <%1> to unmute.").arg(seq)
+                        : tr("[ INFO ]  - Mudlet and game sounds are unmuted. Use <%1> to mute.").arg(seq);
```

Comment:
**I18n issue:** I think you might have a small problem with Far-Eastern locales - their `<` and `>` will likely be translated to `〈` U+3008 {LEFT ANGLE BRACKET} and `〉` U+3009 {RIGHT ANGLE BRACKET} in the translated texts - which are "wide" characters but the ones inserted by the split on (new) line 3037 will be narrow `<` U+003C {LESS-THAN SIGN} and `>` {GREATER-THAN SIGN} which will look rong in those locales because they are "normal" (narrow) characters...

:thinking: actually we might have other places where we are quoting "Normal" Latin1 **punctuation** characters but which may end up needing to be show as "Wide" ones from the Unicode U+3000 to U+303F range.

Reply:
Let's cross that bridge when we get there :slightly_smiling_face: